core/common/src/MapLayerSettings.ts

```diff
@@ -112,34 +112,59 @@ export class MapSubLayerSettings {
   public get idString(): string { return (typeof this.id === "number") ? this.id.toString(10) : this.id; }
 }
 
-/** JSON representation of the settings associated with a map layer.  One or more map layers may be included within a [[MapImageryProps]] object.
+/** JSON representation of the basic settings associated with a map layer.  One or more map layers may be included within a [[MapImageryProps]] object.
  * @see [[MapImageryProps]]
  * @beta
  */
-export interface MapLayerProps {
+export interface MapLayerPropsBase {
   /** Controls visibility of layer. Defaults to 'true'. */
   visible?: boolean;
-  /** Identifies the map layers source.*/
-  formatId: string;
+
   /** Name */
   name: string;
-  /** URL */
-  url: string;
-  /** Source layers. If undefined all layers are displayed. */
-  subLayers?: MapSubLayerProps[];
   /** A transparency value from 0.0 (fully opaque) to 1.0 (fully transparent) to apply to map graphics when drawing,
    * or false to indicate the transparency should not be overridden. Default value: 0.
    * If omitted, defaults to 0. */
   transparency?: number;
   /** True to indicate background is transparent.  Defaults to 'true'. */
   transparentBackground?: boolean;
-  /** Is a base layer.  Defaults to 'false'. */
-  isBase?: boolean;
+
+}
+
+/** JSON representation of the settings for an image based map layer. Extends [[MapLayerPropsBase]] that includes basic settings and adds the image based properties.
+ * @see [[MapImageryProps]]
+ * @see [[ModelMapLayerProps]] for the model map layer properties.
+ * @beta
+ */
+export interface ImageMapLayerProps extends MapLayerPropsBase {
+  /** URL */
+  url: string;
+  /** Identifies the map layers source.*/
+  formatId: string;
+  /** Source layers. If undefined all layers are displayed. */
+  subLayers?: MapSubLayerProps[];
+
   /** Access Key for the Layer, like a subscription key or access token.
    * TODO This does not belong in the props object. It should never be persisted.
```

Comment:
@mdastous-bentley do you have a plan to deal with this (don't store access key)?

Reply:
Great - let's remove it in a separate PR. We don't want it ending up in people's iModels.